apps/generic-issuance-client/src/pages/pipeline/PipelineDetailSection.tsx

```diff
@@ -44,120 +50,216 @@ export function PipelineDetailSection({
+            </SectionContainer>
+          </AccordionPanel>
+        </AccordionItem>
 
         <AccordionItem>
-          <AccordionButton>Latest Logs</AccordionButton>
+          <AccordionButton>Semaphore Groups</AccordionButton>
           <AccordionPanel>
-            <PipelineLatestLogsSection lastLoad={pipelineInfo.lastLoad} />
+            <SectionContainer>
+              <PipelineSemaphoreGroupsSection
+                lastLoad={pipelineInfo.lastLoad}
+              />
+            </SectionContainer>
           </AccordionPanel>
         </AccordionItem>
 
         <AccordionItem>
-          <AccordionButton>Latest Data</AccordionButton>
+          <AccordionButton>Logs</AccordionButton>
           <AccordionPanel>
-            <PipelineLatestDataSection latestAtoms={pipelineInfo.latestAtoms} />
+            <PipelineLatestLogsSection lastLoad={pipelineInfo.lastLoad} />
           </AccordionPanel>
         </AccordionItem>
 
         <AccordionItem>
-          <AccordionButton>Consumers</AccordionButton>
+          <AccordionButton>Data</AccordionButton>
           <AccordionPanel>
-            <PipelineLatestConsumersSection
-              latestConsumers={pipelineInfo.latestConsumers}
+            <PipelineLatestDataSection
+              lastLoad={pipelineInfo.lastLoad}
+              latestAtoms={pipelineInfo.latestAtoms}
             />
           </AccordionPanel>
         </AccordionItem>
 
         <AccordionItem>
-          <AccordionButton>Semaphore Groups</AccordionButton>
+          <AccordionButton>Version History</AccordionButton>
           <AccordionPanel>
-            <PipelineSemaphoreGroupsSection lastLoad={pipelineInfo.lastLoad} />
+            <SectionContainer>
+              <PipelineVersionHistorySection pipelineInfo={pipelineInfo} />
+            </SectionContainer>
           </AccordionPanel>
         </AccordionItem>
 
         {isAdminView && (
-          <AccordionItem>
-            <AccordionButton>
-              Tracing Links&nbsp;<Badge colorScheme="gray">Admin</Badge>
-            </AccordionButton>
-            <AccordionPanel>
-              <UnorderedList>
-                <ListItem>
-                  <PodLink
-                    isExternal={true}
-                    to={getLoadTraceHoneycombLinkForPipeline(pipeline.id)}
-                  >
-                    data load traces {getHoneycombQueryDurationStr()}
-                  </PodLink>
-                </ListItem>
-                <li>
-                  <PodLink
-                    isExternal={true}
-                    to={getAllHoneycombLinkForPipeline(pipeline.id)}
-                  >
-                    all traces related to this pipeline{" "}
-                    {getHoneycombQueryDurationStr()}
-                  </PodLink>
-                </li>
-              </UnorderedList>
-            </AccordionPanel>
-          </AccordionItem>
+          <>
+            {supportsManualTicketTable(pipeline) && isAdminView && (
```

Comment:
I think this should be checking `supportsAddingManualTickets(pipeline)` here, and the inverse on the next accordion item (this error was in the previous code, not something newly introduced here).